src/cli.ts

```diff
@@ -163,7 +163,7 @@ async function contextFromOptions(options: OptionValues): Promise<Context> {
 
 // Network is obtained from context file, if it exists, else from --network flag.
 // This is because ledger does not need a context file
-function networkFromOptions(options: OptionValues): string {
+export function networkFromOptions(options: OptionValues): string {

```

Comment:
please add documentation for all the exported function, interfaces, constants and enums